package.json

```diff
@@ -51,6 +51,7 @@
     "write-file-webpack-plugin": "^4.2.0"
   },
   "dependencies": {
-    "fuse.js": "^3.2.0"
+    "fuse.js": "^3.2.0",
+    "nativesortable": "^0.1.0"
```

Comment:
@alexpdraper This allows the reading list to be sorted using drag and drop moving.